app/main/views/platform_admin.py

```diff
@@ -281,6 +286,50 @@ def platform_admin_letter_validation_preview():
     )
 
 
+@main.route("/platform-admin/clear-cache", methods=['GET', 'POST'])
+@login_required
+@user_is_platform_admin
+def clear_cache():
+    # note: `service-{uuid}-templates` cache is cleared for both services and templates.
+    CACHE_KEYS = {
+        'user': [
+            'user-????????-????-????-????-????????????',
+        ],
+        'service': [
+            'has_jobs-????????-????-????-????-????????????',
+            'service-????????-????-????-????-????????????',
+            'service-????????-????-????-????-????????????-templates',
+            'service-????????-????-????-????-????????????-data-retention',
+            'service-????????-????-????-????-????????????-template-folders',
+        ],
+        'template': [
+            'service-????????-????-????-????-????????????-templates',
+            'template-????????-????-????-????-????????????-version-*',
+            'template-????????-????-????-????-????????????-versions',
+        ],
+        'email_branding': [
+            'email_branding',
+            'email_branding-????????-????-????-????-????????????',
+        ]
+    }
+    form = ClearCacheForm()
+
+    if form.validate_on_submit():
+        to_delete = form.model_type.data
+
+        num_deleted = max(
+            redis_client.delete_cache_keys_by_pattern(pattern)
+            for pattern in CACHE_KEYS[to_delete]
+        )
+
+        flash('Removed {} {} objects from redis'.format(num_deleted, to_delete))
```

Comment:
```suggestion
        flash('Removed {} {} objects from redis'.format(num_deleted, to_delete), 'default')
```
– will make the banner green (good, the thing you’ve asked for has been done) not red (bad, an error has occurred)

Reply:
added in https://github.com/alphagov/notifications-admin/pull/2769/commits/b062a5a13f3b447ed548f416e18e404109301b06